doc/rst/language/spec/ranges.rst

```diff
@@ -872,6 +858,12 @@ It is an error if the count is greater than the ``size`` of the range.
    Each of these ranges represents the ordered set of three indices: 6,
    4, 2.
 
+.. warning::
+
+   The count operator currently treats ranges over ``enum`` or ``bool`` types
+   as bounded on both ends regardless of their ``bounds`` parameters.
+   This might change in the future.
+
```

Comment:
I suspect this is an unstable behavior, and so maybe using the word "unstable" here might be nice.